Util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/Model/BindingProjectModel.cs

```diff
@@ -14,7 +14,13 @@ public class BindingProjectModel
 		public List<MavenArtifactModel> MavenArtifacts { get; set; } = new List<MavenArtifactModel>();
 
 		public string NuGetPackageId { get; set; }
-		public string NuGetVersion { get; set; }
+		public string NuGetVersionBase { get; set; }
+		public string NuGetVersionSuffix { get; set; }
+
+		public string NuGetVersion =>
+			string.IsNullOrWhiteSpace(NuGetVersionSuffix)
+				? NuGetVersionBase
+				: $"{NuGetVersionBase}-{NuGetVersionSuffix}";
```

Comment:
This is technically a breaking change since the version is now readonly, and will change based on the suffix.

Reply:
Also, should we be adding the `-`, or leave it up to the user?

Reply:
I think we leave `-` up to the user.  We need to not apply the suffix for dependencyOnly == true.